doc/docker-dependencies.md

```diff
@@ -0,0 +1,52 @@
+# Docker Dependencies
```

Comment:
> This document doesn't provide any additional insights compared to the `docker-compose` files.

I strongly disagree. It basically replaces the need to add comments to every usage of a docker image in e.g. a `docker-compose` file and then keeping those multiple occurrences up-to-date between each other.

The document explains the thought process behind using specific image versions, such as how @sonallux explained to me why we use OpenJ9 (see [here](https://github.com/jvalue/open-data-service/pull/264#issuecomment-734265654) ).

> But this document will by out of sync in a short time. Especially with detailed version information like "[...] postgrest image in version v7.0.1"

It is true that the document will be out of date if people change used image versions without updating the document, but that would be true for e.g. comments as well.
`PostgREST` does not provide more general docker image tags such as for example `7`, so we have to specify a version to avoid using `latest`.

Reply:
That's why we do have code reviews, to keep those things in sync. We'll give it a shot. If it turns out to not work we still can come up with something different.